bq-connector/README.md

```diff
@@ -0,0 +1,92 @@
+# DocumentAI BQ Connector
+
+## Overview
+The DocAI BQ Connector is a helper library that invokes the DocAI processing library, and formats the response for saving directly into BigQuery.
```

Comment:
Could you provide a diagram or some examples of how exactly the format gets outputted to BQ?

Also, does this work with all processors? Or is it intended just for entity extraction?

Reply:
@deboraelkin2 / @swilliams11 - Could you assist with this question? Which processors would this support?

Reply:
It will work on any processor that returns entities. So for instance, most (all?) of the specialised processors, CDE, CDC, CDS, but wouldn't do much for form processors (except adding some basic metadata to BQ)

Reply:
Ok, @mservidio can you specify this in the readme?